src/libraries/Microsoft.PowerFx.Core/Public/Values/FormulaValueNew.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Public.Types;
+
+namespace Microsoft.PowerFx.Core.Public.Values
+{
+    // Basc new operators. 
+    public partial class FormulaValue
+    {
+        #region Host Utility API
+
+        // Host utility creation methods, listed here for discoverability.
+        // NOT FOR USE IN THE INTERPRETER! When creating new instances in
```

Comment:
If those methods should not be called outside this assembly, can we make them internal?

Reply:
They are public for hosts. Ie, PAD, PVA would call them when creating initial values to pass into fx. 

But the interpreter should *not* call them from within the core of the interpreter.